itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestTimedOutTxnNotificationLogging.java

```diff
@@ -194,6 +191,9 @@ private void runHouseKeeperService() {
   }
 
   private void runTxnHouseKeeperService() {
+    ObjectStore objectStore = new ObjectStore();
```

Comment:
move ObjectStore initialization in setUp

Reply:
Moved them to setup